index.html

```diff
@@ -682,11 +689,7 @@
 && \quad\quad \boldsymbol{\mu}_a \leftarrow \boldsymbol{\mu}_a - \mathrm{min}(\boldsymbol{\mu}_a) \ .
 \end{eqnarray}
 After this shifting, the final extinction coefficient is given by the new $\boldsymbol{\mu}_a + \boldsymbol{\mu}_s$.
-This formulation produces volumetric parameters which reproduce reasonably well the independently specified colors of the transmitted and single-scattered light. Finally the medium phase function anisotropy $g \in [-1, 1]$ is given by **`transmission_scatter_anisotropy`** (the standard Henyey-Greenstein phase function is assumed).
-
-!!! Note: Future work
-   The Henyey-Greenstein phase function is not ideal for representing media with both forward and back scattering lobes (as in the Mie phase function). A richer model such as [#Jendersie2023] could be considered in a future version.
-
+This formulation produces volumetric parameters which reproduce reasonably well the independently specified colors of the transmitted and single-scattered light. Finally the medium phase function anisotropy $g \in [-1, 1]$ is given by **`transmission_scatter_anisotropy`** (the standard Henyey-Greenstein phase function is assumed [^anisotropy_g]).
```

Comment:
Tangential: On the Adobe side we still don't find the interior volume parameterization particularly intuitive, although I'm not sure if there's time to address it for this first version, and I know there are various tradeoffs. It will be interesting to hear how users feel.

Reply:
Sure, of course I'm also uncomfortable having these two different parametrizations in the same model.

That said, it does make some logical sense that the parametrization should differ for the case of very dense and very thin media. The albedo remapping at least cannot be expected to work for very thin translucent volumes, so it seems inappropriate to use that when authoring e.g. colored glass.

(This could have been handled within the framework of a unified volume though, just it was too big a step to make that happen this time round).

Reply:
Yes, I agree with all of that.